src/ugeneui/html/version_news_en.html

```diff
@@ -18,14 +18,34 @@
  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
  MA 02110-1301, USA.
 -->
-<h3>Changes in UGENE v. 39</h3>
+<h3>Changes in UGENE v. 40</h3>
+
+<h4>Alignment Editor</h4>
+<ul>
+    <li style="margin-bottom: 10px;"> Multi-row selection support.</li>
+    <li style="margin-bottom: 10px;">Built Tree dialog now has all PhyML algorithms listed.</li>
```

Comment:
The "Built Tree" dialog now has all PhyML algorithms listed.

Reply:
Fixed. Thank you.